### PR TITLE
Fix notifications

### PR DESCRIPTION
**Pull request by Petitoto**
### Description

- Various fix when calling send_notification_to_user
- Fix notifications title
- Fix CustomTopic to_str() which lead to send wrongly formatted topic names to Titan
- Add get_topit_identifier to get full topic name (with identifiers) the user is subscribed to for a given topic
-  Rename get_topic_membership_XX to plural & add get_topic_memberships_by_user_id_and_topic

### Checklist

- [ ] Created tests which fail without the change (if possible)
- [x] All tests passing
- [ ] Extended the documentation, if necessary

### CODE REVIEW

File: app/cruds/cruds_notification.py [hunk 1]
```diff
@@ -237,7 +237,21 @@ async def get_topic_membership_by_user_id(
     return result.scalars().all()
 
 
-async def get_topic_membership_by_user_id_and_topic(
+async def get_topic_memberships_by_user_id_and_topic(
+    user_id: str,
+    custom_topic: CustomTopic,
```
Comment (armanddidierjean) [inline app/cruds/cruds_notification.py:242]: We could pass a `topic: Topic` parameter

File: app/cruds/cruds_notification.py [hunk 2]
```diff
@@ -237,7 +237,21 @@ async def get_topic_membership_by_user_id(
...
+    result = await db.execute(
+        select(models_notification.TopicMembership).where(
+            models_notification.TopicMembership.user_id == user_id,
+            models_notification.TopicMembership.topic == custom_topic.topic,
+        )
+    )
+    return result.scalars().all()
+
+
+async def get_topic_membership_by_user_id_and_full_topic(
```
Comment (armanddidierjean) [inline app/cruds/cruds_notification.py:254]:
```suggestion
async def get_topic_membership_by_user_id_and_custom_topic(
```

File: app/endpoints/advert.py [hunk 3]
```diff
@@ -282,7 +283,7 @@ async def create_advert(
         )
     except Exception as error:
         hyperion_error_logger.error(
-            f"Error while sending cinema recap notification, {error}"
+            f"Error while sending AMAP recap notification, {error}"
```
Comment (armanddidierjean) [inline app/endpoints/advert.py:286]:
```suggestion
            f"Error while sending advert notification, {error}"
```

Comment (armanddidierjean) [reply app/endpoints/advert.py:286]: I added this try/except, but I'm not sure whether it is really necessary, could this code really fail?

Comment (Petitoto) [reply app/endpoints/advert.py]: For now I think this try except can be useful to monitor if all is going well with notifications

**Pull request merged**